cypress/features/regression/confirm.feature

```diff
@@ -1,154 +1,86 @@
+      | large        | 960              | sizeLarge |
+      | extra-large  | 1080             | sizeExtraLarge |
 
   @positive
   Scenario: Enable background UI
-    When I check enableBackgroundUI checkbox
-      And I open component preview
-    Then Background UI is enabled
+    When I open default "Confirm" component in noIFrame with "confirm" json from "commonComponents" using "enableBackgroundUI" object name
+    Then Background UI is enabled in NoIFrame
 
   @negative
   Scenario: Disable background UI
-    When I check enableBackgroundUI checkbox
-      And I uncheck enableBackgroundUI checkbox
-      And I open component preview
-    Then Background UI is disabled
-
-  @positive
-  Scenario: Disable escape key
-    When I check disableEscKey checkbox
-      And I open component preview
-      And I hit ESC key
-    Then Confirm dialog is visible
-
-  @negative
-  Scenario: Enable escape key
-    When I check disableEscKey checkbox
-      And I uncheck disableEscKey checkbox
-      And I open component preview
-      And I hit ESC key
-    Then Confirm dialog is not visible
-
-  @positive
-  Scenario: Close icon enabled
-    When I check showCloseIcon checkbox
-      And I open component preview
-      And I click close icon
-    Then Confirm dialog is not visible
+    When I open default "Confirm" component in noIFrame with "confirm" json from "commonComponents" using "enableBackgroundUIFalse" object name
+    Then Background UI is disabled in NoIFrame
 
   @negative
   Scenario: Close icon disabled
-    When I check showCloseIcon checkbox
-      And I uncheck showCloseIcon checkbox
-      And I open component preview
-    Then Close icon is not visible
-
-  @positive
-  Scenario: Confirm dialog should dissapear after click onto cancelButton
-    When I open component preview
-      And I click on a cancelButton
-    Then Confirm dialog is not visible
-
-  @positive
-  Scenario: Confirm dialog should dissapear after click onto confirmButton
-    When I open component preview
-      And I click on a confirmButton
-    Then Confirm dialog is not visible
-
-  @positive
-  Scenario: Verify the open action for Confirm dialog
-    Given clear all actions in Actions Tab
-    When I open component preview
-    Then open action was called in Actions Tab
-
-  @positive
-  Scenario: Verify the confirm action for Confirm dialog
-    Given clear all actions in Actions Tab
-      And I open component preview
-    When I click on a confirmButton
-    Then confirm action was called in Actions Tab
-
-  @positive
-  Scenario: Verify the cancel action for Confirm dialog
-    Given clear all actions in Actions Tab
-      And I open component preview
-    When I click on a cancelButton
-    Then cancel action was called in Actions Tab
\ No newline at end of file
+    When I open default "Confirm" component in noIFrame with "confirm" json from "commonComponents" using "enableBackgroundUIFalse" object name
```

Comment:
```suggestion
    When I open default "Confirm" component in noIFrame with "confirm" json from "commonComponents" using "disabledBackgroundUIFalse" object name
```

Reply:
I'm going with convention for the naming: `${knob}${value}`